### PR TITLE
Add requireProto to simplify proto loading

### PR DESCRIPTION
**Pull request by tbetbetbe**
Add requireProto which loads protos using node's simple [module loading](https://nodejs.org/api/modules.html#modules_all_together) system.

This means that services can be delivered as proto files.

### CODE REVIEW

**Comment by tbetbetbe** [comment]
@matthewloring @ofrobots PTAL

**Comment by tbetbetbe** [comment]
@jmuk PTAL

File: test/protobuf.js [hunk 1]
```diff
@@ -94,3 +92,47 @@ describe('`function loadProto(path, [format])`', function() {
...
+  describe('when using an absolute path', function()  {
+    it('should require the proto file without an extension', function() {
+      var isOK = function isOK() {
+        protobuf.requireProto(absolutePath);
+      };
+      expect(isOK).to.not.throw(Error);
+    });
+    it('should require the proto file with an extension', function() {
+      var isOK = function isOK() {
+        protobuf.requireProto(absolutePath);
```
Comment (jmuk) [inline test/protobuf.js]: absolutePath + '.proto'?  Seems exactly same as the previous case.

**Comment by jmuk** [comment]
LGTM, except for one nitpick.

File: test/protobuf.js (hunk 1 above)
Comment (tbetbetbe) [inline test/protobuf.js]: Done

**Comment by tbetbetbe** [comment]
Will merge this PR - the nitpick at LGTM is now resolved

**Pull request merged**